Resources/Prototypes/_NF/Entities/Structures/Shuttles/cannons.yml

```
@@ -13,3 +14,32 @@
   components:
   - type: Machine
     board: Null
+
+# Mail
+- type: entity
+  id: ShuttleGunMailLauncher
+  parent: [ ShuttleGunFriendship ]
+  name: stationary mail launcher
+  description: You've got mail! A small stationary mail launcher that holds 2 grenades.
```

```suggestion
  name: FWD-6500m "Priority"
  description: You've got mail! A small stationary mail launcher that holds 2 mail capsules.
```
We can give this a name if you'd like (and a dumb number because why not).
"FWD" chosen for dumb mail memes, "CC"/"BCC" also good choices if you'd prefer those.
m for "mail", and 6500 because big number?

---

Tried to come up with tech gibberish that could've been abbreviated to "mail BEDAMNED", but gave up on the letter "N." Thanks for the suggestion; it looks better that way.